libcxx/include/tuple

```diff
@@ -382,8 +382,23 @@ struct _LIBCUDACXX_DECLSPEC_EMPTY_BASES __tuple_impl<__tuple_indices<_Indx...>,
     _LIBCUDACXX_CONSTEXPR __tuple_impl()
         _NOEXCEPT_(__all<is_nothrow_default_constructible<_Tp>::value...>::value) {}
 
+    // Handle non-allocator, full initialization
     template <size_t ..._Uf, class ..._Tf,
-              size_t ..._Ul, class ..._Tl, class ..._Up>
+              size_t ..._Ul, class ..._Tl, class ..._Up,
+              _EnableIf<sizeof...(_Uf) == sizeof...(_Tp), bool> = false>
```

Comment:
I know the pack sizes are necessarily the same, but... how do would you feel about comparing `_Tf` with `_Tp`, instead of `_Uf`?

Reply:
Is there a reason to?

Reply:
Ah wait, I just noticed that it's different, but the value would be the same still...

Reply:
It'd then compare the sizes of the type packs, instead of comparing the size of an index pack with a type pack. It doesn't matter semantically; I just wonder if it'd read more nicely with such a change, since it stood out a tiny bit to me. Feel free to disregard this though ;)